spec/features/adventure/user_creates_adventures_specs.rb

```diff
@@ -0,0 +1,87 @@
+require "rails_helper"
+
+feature 'authenticated user creates a an adventure using google maps', %(
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.